src/burnchains/db.rs

```diff
@@ -149,6 +151,8 @@ impl<'a> BurnchainDBTransaction<'a> {
             &u64_to_sql(header.num_txs)?,
             &u64_to_sql(header.timestamp)?,
         ];
+        update_burnchain_height(header.block_height as i64); //promserver
```

Comment:
I don't think this is the right place for this.  I think this should happen in the top-level `Burnchain::sync()` method.  (Really, I think this should happen in the chains coordinator, but the chains coordinator doesn't drive burnchain synchronization yet).

Reply:
This has been moved to `src/burnchains/burnchain.rs`